includes/elements/container.php

```diff
@@ -338,6 +344,7 @@ protected function register_container_layout_controls() {
 				Breakpoints_Manager::BREAKPOINT_KEY_LAPTOP => $min_affected_device,
 				Breakpoints_Manager::BREAKPOINT_KEY_TABLET_EXTRA => $min_affected_device,
 				Breakpoints_Manager::BREAKPOINT_KEY_TABLET => $min_affected_device,
+				Breakpoints_Manager::BREAKPOINT_KEY_MOBILE_EXTRA => $min_affected_device,
```

Comment:
I know this isn't you who did it, but I do not understand why create the `$width_control_settings` then use `array_merge()` in the `add_responsive_control()` call.
Just put everything in the `add_responsive_control()` please.

Reply:
@udidol 

I believe the logic makes sense. 

The width controls are used both for the boxed width and for the full width version. The `array_merge()` makes sense, because in this way we only have to define the controls once, and they can be used twice.

Reply:
I didn't see it used twice, OK.